R/tm_outliers.R

```diff
@@ -269,9 +269,9 @@ srv_outliers <- function(id, data, reporter, filter_panel_api, outlier_var,
       select_validation_rule = list(
         outlier_var = shinyvalidate::compose_rules(
           shinyvalidate::sv_required("Please select a variable"),
-          rule_diff("categorical_var")
+          rule_diff("categorical_var", "Categorical factor", "Variable")
```

Comment:
When we wrote rule functions we tried to have them have as few arguments as possible. Since this function is only ever used to test those two variables, the message can be hard coded.

Reply:
Oh, even more so it was to enforce that the message is the same because now you will get two messages for what is basically the same error, whereas if the messages are identical, duplicates are removed.

See `?teal::validate_inputs`.

Reply:
Ok, I'll rollback those arguments :+1: